packages/sdk/tsconfig.build.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.build.json",
   "compilerOptions": {
     "outDir": "dist",
-    "emitDeclarationOnly": true
+    "emitDeclarationOnly": true,
+    "paths": {}
```

Comment:
What does that do? Maybe add a comment here, it's allowed in tsconfig

Reply:
We only want to use `paths` for the build step if we are bundling. So all non-bundled packages have an empty one. We don't want `esbuild` to build via the paths, because it is a clear and separate dependency. Additionally, for building types, it also adds friction because TS would include all the types from the paths into `dist` instead of using the ones from `node_modules`.

Reply:
I think the tl;dr; is used an empty `path` is you're not bundling.

Reply:
Do you think it would make sense to have a separate ts preset for non-bundled packages, with those 2 options and this comment, so we don't have to repeat them everywhere?

Reply:
Definitely, we could have at the root `tsconfig.build.bundle.json` and `tsconfig.build.regular.json` that is extended in the package `tsconfig.build.json`.